portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,91 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>more about B</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>a page about me</h1>
+      <p>this is my portfolio, a little more about B</p>
     </div>
+
+    <div id="body-content">
+        <div id="about" class="sidebar">
+            <div id="links" class="box">
+                <p2>a crash course in me</p2>
```

Comment:
did you mean `<h2>` here? I don't think i've seen `<p2>` before.

Reply:
chris commented the same about <pN> on my other pull requests, i use them to access that particular paragraph in the css file. i can see how this can get pretty confusing when your html page gets really long, i will read up on how to style the paragraphs without having to do <pN>